libraries/AP_Scripting/examples/message_interval.lua

```diff
@@ -5,17 +5,50 @@
 -- during the initial connection, so by resetting them periodically we ensure we get
 -- the expected telemetry rate
 
+local MAV_SEVERITY = {EMERGENCY=0, ALERT=1, CRITICAL=2, ERROR=3, WARNING=4, NOTICE=5, INFO=6, DEBUG=7}
+
+local loop_time = 5000 -- number of ms between runs
+
+-- mavlink message ids
+local AHRS_ID                   = uint32_t(163)
+local AHRS2_ID                  = uint32_t(178)
+local ATTITUDE_ID               = uint32_t(30)
+local EKF_STATUS_REPORT_ID      = uint32_t(193)
+local GLOBAL_POSITION_INT_ID    = uint32_t(33)
+local GPS_RAW_INT_ID            = uint32_t(24)
+local LOCAL_POSITION_NED_ID     = uint32_t(32)
+local NAV_CONTROLLER_OUTPUT_ID  = uint32_t(62)
+local POWER_STATUS_ID           = uint32_t(125)
+local RAW_IMU_ID                = uint32_t(27)
+local RC_CHANNELS_ID            = uint32_t(65)
+local SERVO_OUTPUT_RAW_ID       = uint32_t(36)
+local SYS_STATUS_ID             = uint32_t(1)
+local SYSTEM_TIME_ID            = uint32_t(2)
+local VFR_HUD_ID                = uint32_t(74)
+local VIBRATION_ID              = uint32_t(241)
+local WIND_ID                   = uint32_t(168)
+
+-- serial port
+local serial_port = 0
+
 -- intervals is a table which contains a table for each entry we want to adjust
 -- each entry is arranged as {the serial link to adjust, the mavlink message ID, and the broadcast interval in Hz}
-local intervals = {{0, uint32_t(30), 2.0},  -- First serial, ATTITUDE, 2Hz
-                   {0, uint32_t(163), 5.0}} -- First serial, AHRS, 5Hz
+local intervals = {{serial_port, ATTITUDE_ID, 2.0}, -- ATTITUDE, 2Hz
+                   {serial_port, AHRS_ID, 1.0},     -- AHRS, 5Hz
```

Comment:
Comment is out of date

Reply:
Thanks for noticing that.  I've updated the rate to match the comment, they're just examples in any case, for the payload the table is filled in differently (everything is off except global-position-int)